cypress/e2e/docs-smoke-test/site-overrides.cy.ts

```diff
@@ -0,0 +1,28 @@
+import { URL_DOCS_SMOKE_TEST } from '../../support/urls';
+
+const overridesPageUrl = `${URL_DOCS_SMOKE_TEST}views/overrides`;
+
+describe('Site overrides', () => {
+  beforeEach(() => {
+    cy.visit(overridesPageUrl);
+  });
+  describe('when the page is rendered', () => {
+    it('should render "Custom Site Title" as site title and custom link in the sidebar', () => {
+      cy.get('span[id="site-title"]').contains('Custom Site Title');
+      cy.get('span#site-title')
+        .closest('a')
+        .should('have.attr', 'href')
+        .and('include', '/views/custom-anchor');
```

Comment:
Just a question where doesnt this '/views/custom-anchor' get defined in the test setup?